test/e2e/e2e_test.go

```diff
@@ -207,8 +210,11 @@ func createAndApproveSignup(t *testing.T, awaitility *wait.Awaitility, username
 func postSignup(t *testing.T, route string, identity authsupport.Identity) {
 	// Call signup endpoint with a valid token.
 	emailClaim := authsupport.WithEmailClaim(uuid.NewV4().String() + "@email.tld")
+	givenNameClaim := authsupport.WithGivenNameClaim("jane")
+	familyNameClaim := authsupport.WithFamilyNameClaim("doe")
+	companyClaim := authsupport.WithCompanyClaim("red hat")
```

Comment:
Instead of hardcoding the same name for all the usersignups in the tests we could generate unique ones based on username or userID. Something like:
```
givenNameClaim := authsupport.WithGivenNameClaim(identity.Username + "-First-Name")
```

Reply:
fixed here: bb2c80307a4f90e88a7cc77e78ee01a1a7dbbe78